scripts/pos/validator_registration_utils.go

```diff
@@ -0,0 +1,33 @@
+//go:build relic
+
+package main
+
+import (
+	"fmt"
+	"github.com/deso-protocol/core/bls"
+	"github.com/deso-protocol/core/lib"
+)
+
+func getBLSVotingAuthorizationAndPublicKey(blsKeyStore *lib.BLSKeystore) (*bls.PublicKey, *bls.Signature) {
+	pubKey := blsKeyStore.GetSigner().GetPublicKey()
+	votingAuthPayload := lib.CreateValidatorVotingAuthorizationPayload(pubKey.ToBytes())
```

Comment:
Just realized... the `CreateValidatorVotingAuthorizationPayload` function is supposed to take in the transactor's ECDSA public key bytes. Seems like we need to have the user enter their validator's pub key as a string too in the main function
```suggestion
func getBLSVotingAuthorizationAndPublicKey(blsKeyStore *lib.BLSKeystore, transactorPubKey *lib.PublicKey) (*bls.PublicKey, *bls.Signature) {
	pubKey := blsKeyStore.GetSigner().GetPublicKey()
	votingAuthPayload := lib.CreateValidatorVotingAuthorizationPayload(pubKey.ToBytes())
```